### PR TITLE
For #2794 - Adjust engine margins when enter/exit fullscreen

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [X] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt [hunk 1]
```diff
@@ -388,6 +390,21 @@ class BrowserFragment : Fragment(), BackHandler, CoroutineScope {
         }
     }
 
+    private fun changeEngineMargins(swipeRefresh: View, inFullScreen: Boolean) {
+        swipeRefresh.apply {
+            val toolbarAndQASSize = resources.getDimension(R.dimen.toolbar_and_qab_height).toInt()
+            val toolbarSize = resources.getDimension(R.dimen.browser_toolbar_height).toInt()
+            (layoutParams as CoordinatorLayout.LayoutParams).apply {
+                setMargins(
```
Comment (jonalmeida) [inline app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: I find this logic hard to follow having known the context that it came from, so it might be harder for someone in the future to follow it as well.

Instead of inlining the logic in the place that it's being set, how about separate that logic outside, and then have a simple single setMargin with the final values?

[`DisplayToolbar`](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt#L300-L303) has an example of separating out the logic with a when block:

```kotlin
val (topMargin, bottomMargin) = when {
 // logic statements here.
}
setMargin(0, topMargin, 0, bottomMargin)
```

Just a suggestion, maybe this doesn't end up simplifying anything!

Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: Good call! I had never returned pairs like that, much cleaner!

File: app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt [hunk 2]
```diff
@@ -388,6 +390,26 @@ class BrowserFragment : Fragment(), BackHandler, CoroutineScope {
         }
     }
 
+    private fun changeEngineMargins(swipeRefresh: View, inFullScreen: Boolean) {
+        swipeRefresh.apply {
+            val toolbarAndQASSize = resources.getDimension(R.dimen.toolbar_and_qab_height).toInt()
```
Comment (jonalmeida) [inline app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: nit: QASSize? QABSize?
